vendor/bat-native-ledger/src/bat/ledger/internal/request/attestation_requests.cc

```diff
@@ -0,0 +1,61 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+
+#include "base/strings/stringprintf.h"
+#include "bat/ledger/internal/static_values.h"
+#include "bat/ledger/internal/request/attestation_requests.h"
+#include "bat/ledger/internal/request/request_util.h"
+
+namespace braveledger_request_util {
+
+std::string GetStartAttestationDesktopUrl() {
+  return BuildUrl("/captchas", PREFIX_V1, ServerTypes::kPromotion);
+}
+
+std::string GetCaptchaUrl(const std::string& captcha_id) {
+  const std::string path = base::StringPrintf(
+      "/captchas/%s.png",
+      captcha_id.c_str());
+  return BuildUrl(path, PREFIX_V1, ServerTypes::kPromotion);
+}
+
+std::string GetClaimAttestationDesktopUrl(const std::string& captcha_id) {
+  const std::string path = base::StringPrintf(
+      "/captchas/%s",
```

Comment:
I see a lot of strings all around. No issue, just a good practice to have all strings as constants if they ever need changing